From c46fbc8c0b4014cc4a66e6b411a8ad4c13f19f2e Mon Sep 17 00:00:00 2001
From: unman <unman@thirdeyesecurity.org>
Date: Fri, 26 Feb 2021 04:48:45 +0000
Subject: [PATCH] Brand with Qubes

---
 driver/XScreenSaver.ad.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/driver/dialog.c b/driver/dialog.c
index e5834b6..666fb5f 100644
--- a/driver/dialog.c
+++ b/driver/dialog.c
@@ -1047,13 +1047,13 @@ window_init (Widget root_widget, int splash_p)
     *s = 0;
     ws->heading_label = (char *) malloc (100);
     ws->version = strdup(version);
-    sprintf (ws->heading_label, "XScreenSaver %.4s, v%.10s", year, version);
+    sprintf (ws->heading_label, "Qubes OS 4.1");
 
     if (splash_p)
       {
         ws->body_label = (char *) malloc (100);
         sprintf (ws->body_label,
-           _("Copyright \xC2\xA9 1991-%.4s by\nJamie Zawinski <jwz@jwz.org>"),
+           _("Copyright \xC2\xA9 1991-%.4s by\nJamie Zawinski"),
                   year);
       }
   }
