From c46fbc8c0b4014cc4a66e6b411a8ad4c13f19f2e Mon Sep 17 00:00:00 2001
From: unman <unman@thirdeyesecurity.org>
Date: Fri, 26 Feb 2021 04:48:45 +0000
Subject: [PATCH] Brand with Qubes

---
 driver/XScreenSaver.ad.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/driver/XScreenSaver.ad.in b/driver/XScreenSaver.ad.in
index 828e32b..f7d3546 100644
--- a/driver/XScreenSaver.ad.in
+++ b/driver/XScreenSaver.ad.in
@@ -316,15 +316,15 @@ XScreenSaver.bourneShell:		/bin/sh
 
 ! Static text in the dialog boxes:
 !
-*passwd.heading.label:		XScreenSaver %s
+*passwd.heading.label:		Qubes OS 4.0
 *passwd.body.label:		This screen is locked.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
 
-*splash.heading.label:		XScreenSaver %s
+*splash.heading.label:		Qubes OS 4.0
 *splash.body.label:		Copyright \251 1991-2020 by
-*splash.body2.label:		Jamie Zawinski <jwz@jwz.org>
+*splash.body2.label:		Jamie Zawinski 
 *splash.demo.label:		Settings
 *splash.help.label:		Help
 
--
libgit2 1.0.1

